authors/apps/articles/migrations/0001_initial.py

```diff
@@ -23,12 +28,19 @@ class Migration(migrations.Migration):
                 ('article_slug', models.SlugField(editable=False, max_length=255, unique=True)),
                 ('created_at', models.DateTimeField(auto_now_add=True)),
                 ('updated_at', models.DateTimeField(auto_now=True)),
+                ('author', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='authorshaven', to=settings.AUTH_USER_MODEL)),
+                ('dislike', models.ManyToManyField(blank=True, related_name='dislike', to=settings.AUTH_USER_MODEL)),
```

Comment:
Make sure only the migrations you have caused as a result of changes in this PR are present in the migrations file